Bottom: 335652c037c611042518784cfbb9c4fadce8d782
Top:    a63ffccfe8f6659d4b0c4d83aecf6f9c0f7c94fb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 15:17:52 +0100

Remove SWAP_IDENT, pass it as an argument where needed

Magic state is bad ;)


---

diff --git a/src/macro.h b/src/macro.h
index 19f80b8..95236af 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -34,11 +34,7 @@
 
 /* ---------------------------------------------------------------------------
  * macros to transform coord systems
- * draw.c uses a different definition of TO_SCREEN
  */
-#ifndef	SWAP_IDENT
-#define	SWAP_IDENT			Settings.ShowSolderSide
-#endif
 
 #define	SWAP_SIGN_X(x)		(x)
 #define	SWAP_SIGN_Y(y)		(-(y))
